config/locales/en.yml

```diff
@@ -5,7 +5,7 @@ en:
     language: language
   site-header:
     home: Home
-    sign-in: Sign In
+    sign-in: Create Account / Sign In
```

Comment:
Maybe this is a discussion with the rest of the group, but maybe we go with "Register"?

Reply:
Good idea. Will change this. And then can change later, if needed, once we've had a chance to discuss with team.

I think the whole header needs a bit of a rethink - was going to add the user's display name, but there's no obvious place. Maybe as a user dropdown (will try to use this actually).

Reply:
Actually, I add that second dropdown in the PR im working on, right now I have it as:

* Username
  * My Profile
  * My Subscriptions
  * Sign Out

Reply:
Sounds good! I'll need this for my subscriptions work – could you add that as a commit to this branch and push it up?

Reply:
Sure, can do that right now actually

Reply:
Done 😃

Reply:
Nice one!